src/features/board/Board.tsx

```diff
@@ -15,11 +42,23 @@ export function Board () {
             <button onClick={() => dispatch(attack([1, 0]))}> Dragon attack Mage</button>
             <button onClick={() => dispatch(attack([1, 1]))}> Dragon attack Vampire</button>
             <div>
-                { opponentCards.map((card, i) => <Card key={`${card}${i}`} cardProperties={card} />) }
+                { opponentCards.map((card, i) => (
+                    <div>
+                        <Card key={`${card}${i}`} cardProperties={card} />
+                    </div>
+                )) }
             </div>
             <hr/>
             <div>
-                { playerCards.map((card, i) => <Card key={`${card}${i}`} cardProperties={card} />) }
+                { playerCards.map((card, i) => (
+                    <div
+                        onClick={() => { handleSelectCard(i) }}
+                        onContextMenu={handleContextMenu}
```

Comment:
I did not know this event :D 

https://developer.mozilla.org/fr/docs/Web/API/Element/contextmenu_event

Its kind of ugly to have to preventDefault it but i think you have no choice :p